client/src/Invitation.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+export default function FormDialog() {
+  const useStyles = makeStyles((theme) => ({
+    margin: {
+      margin: theme.spacing(3),
+    },
+    extendedIcon: {
+      marginRight: theme.spacing(1),
+    },
+  }));
+  const classes = useStyles();
+  const inputRef = React.createRef();
+  const [emails, setEmail] = React.useState("");
+  const [open, setOpen] = React.useState(false);
+  const [uniqueID, setID] = React.useState(
+    "https://www.EKLN-messenger.com/join/" +
+      (Math.floor(Math.random() * 90000000) + 10000000)
+  );
+  const [emailList, setEmailList] = React.useState([]);
+
+  /*generate unique id for URL */
```

Comment:
Kevin's model plans to use the invitation id from the database for this, so we'll need to sync about how to handle this